test/libsolidity/smtCheckerTests/operators/assignment_contract_member_variable_array_3.sol

```diff
@@ -0,0 +1,21 @@
+pragma experimental SMTChecker;
+contract A {
+	int[] a;
+	function f() public {
+		require(a.length == 1 && a[0] == 1);
+		int[] storage u = a;
+		assert(u[0] == 1); // should hold
+		int[] memory b = new int[](2);
+		a = b;
+		assert(u[0] == 1); // should fail
+		A.a = b;
+		assert(u[0] == 1); // should fail
```

Comment:
@blishko so this used to not fail before this PR?

Reply:
In the latest release one of them is detected and one is not.
After the fix #11088, actually not of these two failures is detected.